deepchecks/vision/utils/image_functions.py

```diff
@@ -55,6 +56,28 @@ def numpy_to_image_figure(data: np.ndarray):
     return go.Image(z=data, hoverinfo='skip')
 
 
+def plot_image_png(data: np.ndarray, labels=None):
+    dimension = data.shape[2]
+    if dimension == 1:
+        fig = go.Figure(go.Heatmap(z=data.squeeze(), colorscale='gray', hoverinfo='skip'))
+        apply_heatmap_image_properties(fig)
+        fig.update_traces(showscale=False)
+    elif dimension == 3:
+        fig = go.Figure(go.Image(z=data, hoverinfo='skip'))
+    else:
+        raise DeepchecksValueError(f'Don\'t know to plot images with {dimension} dimensions')
+
+    if labels:
+        label_bbox_add_to_figure(labels, fig)
+
+    fig.update_yaxes(showticklabels=False, visible=True, fixedrange=True, automargin=True)
+    fig.update_xaxes(showticklabels=False, visible=True, fixedrange=True, automargin=True)
+    fig.update_layout(margin={'l': 0, 'r': 0, 't': 0, 'b': 0})
+    png = base64.b64encode(fig.to_image('png')).decode('ascii')
```

Comment:
🤢

Reply:
Does it mean we don't need kaleido anymore?

Reply:
what's the problem?

Reply:
no, plotly uses kaleido for the export